package.json

```diff
@@ -18,7 +18,8 @@
   },
   "scripts": {
     "test": "mocha --reporter-option maxDiffSize=0",
-    "test:cov": "nyc npm run test"
+    "test:cov": "nyc npm run test",
+    "lint": "eslint ./src"
```

Comment:
We may want our tests to be linted too. Not sure if there's something stopping this?

Reply:
Just the "no-udef" rule, but I guess we can disable it for this file.

Reply:
It might work if we add mocha and chai to the eslint env?